### PR TITLE
[wg/fedid] Update from PR to beyond CR

### PR DESCRIPTION
**Pull request by simoneonofri**
As per https://github.com/w3c/charter-drafts/issues/566 by @frivoal

[cc @wseltzer @hlflanagan]

### CODE REVIEW

File: 2024/wg-fedid.html [hunk 1]
```diff
@@ -292,8 +292,7 @@ <h2>Success Criteria</h2>
       <!-- Testing and interop -->
       <p>In order to advance
         to
```
Comment (frivoal) [inline 2024/wg-fedid.html]:
```suggestion
```
I don't think this word is necessary after the other changes

**Review by frivoal — APPROVED** [review]
Thanks. I think this is a useful change, and suggest adopting it in the charter template as well.

File: 2024/wg-fedid.html [hunk 2]
```diff
@@ -291,9 +291,7 @@ <h2>Success Criteria</h2>
 
       <!-- Testing and interop -->
       <p>In order to advance
-        to
-        <a href="https://www.w3.org/policies/process/#RecsPR" title="Proposed Recommendation">Proposed
-          Recommendation</a>, each normative specification is expected to have
+        <i>beyond</i> <a href="https://www.w3.org/policies/process/#RecsCR" title="Candidate Recommendation">Candidate Recommendation</a>, each normative specification is expected to have
```
Comment (tantek) [inline 2024/wg-fedid.html]:
```suggestion
        <i>beyond</i> <a href="https://www.w3.org/policies/process/#RecsCR">Candidate Recommendation</a>, each normative specification is expected to have
```

**Review by tantek — CHANGES_REQUESTED** [review]
Looks good except suggest same change that @jyasskin suggested in #578, no need to duplicate element content in title attribute.

**Review by tantek — APPROVED** [review]

**Pull request merged**